### DISCUSSION TITLE
Suricata Stun rule for Microsoft Teams

### DISCUSSION THREAD

**Discussion by tommorgan365** [Security-Onion-Solutions/securityonion#1684] [Unsupported Versions] [answered]
Stun rule for Microsoft Teams is firing.  What is the best approach modify the rule or exclude teams public ip's ?

**Comment by tommorgan365** [comment]
Also, 
Do you have an example of using a regex in the modify section of IDSTOOLS -> SIDS -> modify ?

**Comment by TOoSmOotH** [comment, marked as answer]
Check this thread. I would use thresholds before I used modify.

https://github.com/Security-Onion-Solutions/securityonion/discussions/1631
